content/config.yml

```diff
@@ -1012,7 +744,10 @@ location: 'blog'
 author: 'oddbird'
 comments: false
 
-blog_title: 'Our Odd Thoughts...'
+# blog_title:
+#   - title: 'The Blog'
+#     type: 'Articles'
+#     tagline: 'Our Odd Thoughts...'
```

Comment:
Yeah, Sondra and I weren't sure it was needed (or good-looking) at this point. I'm open either way.